### DISCUSSION TITLE
Asynchronous send messages

### DISCUSSION THREAD

**Discussion by mehrdad2000** [tilln/jmeter-iso8583#99] [General]
Hi
Is there any way to send asynchronous message with iso8583 without wait for reply?
Thanks

**Comment by tilln** [comment]
You can set the Sampler "Timeout" to 0.
Refer [documentation](https://github.com/tilln/jmeter-iso8583?tab=readme-ov-file#response-validation):

> Timeout (ms): A response timeout in milliseconds can be defined individually for each sampler. The value 0 can be used to send requests without expecting any response message (["fire and forget"](https://www.enterpriseintegrationpatterns.com/patterns/conversation/FireAndForget.html); since v1.1).
